pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1041,8 +1042,19 @@ func (ctrl *PersistentVolumeController) reclaimVolume(volume *v1.PersistentVolum
 	case v1.PersistentVolumeReclaimDelete:
 		glog.V(4).Infof("reclaimVolume[%s]: policy is Delete", volume.Name)
 		opName := fmt.Sprintf("delete-%s[%s]", volume.Name, string(volume.UID))
+		startTime := time.Now()
 		ctrl.scheduleOperation(opName, func() error {
-			return ctrl.deleteVolumeOperation(volume)
+			err := ctrl.deleteVolumeOperation(volume)
```

Comment:
You can return volume plugin name from `deleteVolumeOperation` so you don't need to look it up below.

Reply:
Done. I also removed the `storage_class` as dimension because it seems that it's not very useful in this case (and to avoid making it more complex).